.github/workflows/ci.yml

```
@@ -101,6 +164,7 @@ jobs:
     name: Linter
     runs-on: ubuntu-latest
     needs: generate-json-schema
+    if: needs.get-diff.outputs.output-contract == 'true' || needs.get-diff.outputs.output-certora-contract == 'true' || needs.get-diff.outputs.output-scripts == 'true' || needs.get-diff.agents.output-scripts == 'true'
```

```suggestion
    if: needs.get-diff.outputs.output-contract == 'true' || needs.get-diff.outputs.output-certora-contract == 'true' || needs.get-diff.outputs.output-scripts == 'true' || needs.get-diff.outputs.output-agents == 'true'
```

Didn't expect typos here :(

---

Done